### PR TITLE
Build home page

### PR DESCRIPTION
**Pull request by ishitakeshawani**
- added categories to the home page
- added quiz context 
- added a quiz reducer to get categories
- preview link: https://inspiring-clafoutis-38d682.netlify.app/

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
 
 function App() {
   return (
     <div className="App">
-     <HomePage />
+      <NavBar />
+      <Routes>
+        <Route path="/" element={<HomePage />} />
+      </Routes>
     </div>
```
Comment (aniketsingh13) [inline src/App.js]: you can put it in the seperate file

**Review by aniketsingh13 — APPROVED** [review]
ui look good and code structure is also good

File: .env [hunk 2]
```diff
@@ -0,0 +1 @@
+REACT_APP_JWT_SECRET=cbvjsbdfhdsjkfnsthis
```
Comment (rohanmathur91) [inline .env]: you should not push `.env` to github, remove it

File: src/contexts/quiz-context.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        const {
+          data: { categories },
+        } = await axios.get("/api/categories");
+
+        dispatch({
+          type: "SET_CATEGORY",
+          payload: categories,
+        });
+      } catch (error) {
+        console.log(error);
```
Comment (rohanmathur91) [inline src/contexts/quiz-context.js]: show the error via some toast

**Review by rohanmathur91 — APPROVED** [review]
- UI looks good and clean
- Update the footer links
- PR looks good to merge

File: src/App.js (hunk 1 above)
Comment (ishitakeshawani) [reply src/App.js]: done

File: .env (hunk 2 above)
Comment (ishitakeshawani) [reply .env]: yes

File: src/contexts/quiz-context.js (hunk 3 above)
Comment (ishitakeshawani) [reply src/contexts/quiz-context.js]: done

**Pull request merged**